### PR TITLE
fix(select): Remove blue background in IE on focus

### LINKED ISSUE #3496: Select value is highlighted in IE on focus
### What MDC Web Version are you using?

0.39.1

### What browser(s) is this bug affecting?

IE 11 💩

### What OS are you using?

Windows 10

### What are the steps to reproduce the bug?

1. View the [mdc-select screenshot test page](https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2018/08/28/00_57_26_335/spec/mdc-select/issues/3230.html) in IE 11

### What is the expected behavior?

Select value should have _black_ text and a _transparent_ background when focused:

image

### What is the actual behavior?

Select value has _white_ text (`#ffffff`) and a _blue_ background (`#3a9bfc`) when focused:

image

Sometimes, just for fun, IE renders a _slightly darker blue_ (`#137ad4`):

image

### Any other information you believe would be useful?

The flaky blue background color (`#3a9bfc` vs. `#137ad4` on an Apple Thunderbolt Display) \
causes spurious screenshot diffs. E.g.:

* PR #3501:
    - [Report `2018-09-03T00:03:31.368Z`](https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2018/09/03/00_03_31_229/report/report.html)
    - [Report `2018-09-03T00:40:58.983Z`](https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2018/09/03/00_40_58_861/report/report.html)
    - [Report `2018-09-03T01:11:04.043Z`](https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2018/09/03/01_11_03_898/report/report.html)
* PR #3412:
    - [Report `2018-08-31T23:16:24.685Z`](https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2018/08/31/23_16_24_535/report/report.html)
* PR #3434:
    - [Report `2018-09-04T03:26:55.129Z`](https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2018/09/04/03_26_54_981/report/report.html)

Fixing it should be as simple as:

```css
.mdc-select__native-control::-ms-value {
  background-color: transparent;
  color: inherit;
}
```

### PR DESCRIPTION
**Pull request by acdvorak**
Fixes #3496

### CODE REVIEW

File: test/screenshot/spec/mdc-select/issues/3496.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    <link rel="stylesheet" href="../../../out/mdc.typography.css">
+    <link rel="stylesheet" href="../../../out/spec/fixture.css">
+    <link rel="stylesheet" href="../../../out/spec/mdc-select/fixture.css">
+  </head>
+
+  <body class="test-container mdc-typography">
+    <main class="test-viewport test-viewport--mobile">
+      <div class="test-layout">
+
+        <!-- This tests that select suppresses IE's default blue background on focus -->
```
Comment (williamernest) [inline test/screenshot/spec/mdc-select/issues/3496.html]: I don't think we need a new screenshot page for this. We can add these comments to the 3230.html page stating that the first select is verifying both issues. Maybe rename the file `3230_3496.html`?

Comment (acdvorak) [reply test/screenshot/spec/mdc-select/issues/3496.html]: Done

**Comment by kfranqueiro** [comment]
Why is there a diff in IE and not Edge? Shouldn't these changes affect both? Or does it have to do with the browser's default styles?

File: packages/mdc-select/mdc-select.scss [hunk 2]
```diff
@@ -66,6 +66,11 @@
       display: none;
     }
 
+    &::-ms-value {
+      background-color: transparent;
+      color: inherit;
```
Comment (kfranqueiro) [inline packages/mdc-select/mdc-select.scss]: Would replacing this with `@include mdc-select-ink-color($mdc-select-ink-color);` obviate the need for #3513?

**Comment by acdvorak** [comment]
It's due to the browser's default styles. Earlier versions of Edge had the same problem, but they fixed it around version 15 or 16 I think.

**Comment by williamernest** [comment]
Because `mdc-typography` was added to the body.

**Pull request merged**